lib/rules/validate-indentation.js

```diff
@@ -66,11 +93,25 @@ module.exports = function() {};
 module.exports.prototype = {
 
     configure: function(validateIndentation) {
-        assert(
-            validateIndentation === '\t' ||
-                (typeof validateIndentation === 'number' && validateIndentation > 0),
-            'validateIndentation option requires a positive number of spaces or "\\t"'
-        );
+        this._includeEmptyLines = false;
+
+        if (typeof validateIndentation === 'object') {
+            assert(
```

Comment:
figure out how to avoid the duplicate assertion here... perhaps:

```
this._includeEmptyLines = (validateIndentation.includeEmptyLines === true);
validateIndentation = validateIndentation.value;
```

dropping the else, and then just asserting after?